.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,45 @@
+name: Test quickstart.sh On Different Distros
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  test-amazon-linux:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the AmazonLinux Docker image
```

Comment:
Have you tested this out? Do we not need setup docker on the github workers anymore? This could be the case since they have pushed a bunch of new tools on the base image.

Reply:
See the section below, all 4 tests have passed.

The github workers `runs-on: ubuntu-latest`, and the Dockfile specifies the base image `FROM amazonlinux:latest` (https://github.com/superblocksteam/agent/blob/main/scripts/distro-test/amzn/Dockerfile#L1)

So my understanding is that, we don't need to set github worker to run on `amazonlinux`, am I right?